pages/instructorQuestionManualGrading/instructorQuestionManualGrading.js

```diff
@@ -0,0 +1,114 @@
+const ERR = require('async-stacktrace');
+const express = require('express');
+const router = express.Router();
+const path = require('path');
+const debug = require('debug')('prairielearn:' + path.basename(__filename, '.js'));
+const { error, sqlDb, sqlLoader} = require('@prairielearn/prairielib');
+
+const sql = sqlLoader.loadSqlEquiv(__filename);
+
+class NoSubmissionError extends Error {
+    constructor(message) {
+      super(message);
+      this.name = 'NoSubmissionError';
+    }
+  }
+
+// TODO:
+// eslint-disable-next-line no-unused-vars
+router.get('/', (req, res, next) => {
+    console.log(res.locals);
+    const params = {instance_question_id: res.locals.instance_question_id};
```

Comment:
From my perspective, I'm a bit torn if this should actually depend upon an instance ID or a grading job ID, i.e. if we treat querying manual grading jobs as a "queue" from the grading jobs table, or we just let people always try to access by submission ID.

This does work, and it makes equal sense, so I'm comfortable building off of this logic here.

Reply:
I think that's a good question to consider. What would you prefer? 

If we navigate to an assessment instance and then select a question, we can load the latest instance_question on an assessment ID. If we go purely by grading_job ID, then we just need to figure out the front-end flow and what leads up to the query. We need the correct front-end UI params. 

We also likely need an easy way to increment through each of the instance_questions of the assessments, as they are being marked.

Reply:
I think it makes logical sense either way, so I'm comfortable with working with the current queries.

Reply:
Yeah, I think if this were to change, the payload and data on the front-end would be the same regardless... so at least this allows you to work.